src/rust/engine/fs/src/directory.rs

```diff
@@ -128,11 +180,11 @@ impl Directory {
     }
   }
 
-  pub fn as_directory(&self) -> remexec::Directory {
-    self.tree.as_directory()
+  pub fn as_remexec_directory(&self) -> remexec::Directory {
```

Comment:
Bike shed: Would `reapi` be better? `remexec` was my not-very-thought-out choice for the `use` of the module with the protos in it, and not as part of the API names.

Reply:
`remexec` is what is already in ~every file dealing with that API. Can change it when we change the other usages.